main/cmds_test.go

```diff
@@ -122,9 +122,8 @@ func Test_runCmd_withTestFile(t *testing.T) {
 
 	// print files in directory
 	files, err := ioutil.ReadDir(dir)
-	if err != nil {
-		t.Log(err)
-	}
+	t.Log(err)
```

Comment:
I think the better pattern is

```golang
if err != nil {
    t.Fail(err)
}
```

You only log in the case of an error, and if there is an error it is fatal to the test.

Reply:
oh, I see. thank you!

Reply:
Fail does not take any parameters

Reply:
nvm, fatal works